content/en/docs/KRE/user/60_kais_runner_sdk/_index.md

```diff
@@ -0,0 +1,266 @@
+---
+title: "KAI Server Runner SDK"
+linkTitle: "KAI Server Runner SDK"
+weight: 60
+description: >
+  Code your nodes on top of the KAI Server Runner SDK
+---
+
+
+## The Runner SDK
+
+KAI Server supports Python and Golang providing an SDK that allows users to receive,
+process and send messages between nodes.
+
+Thanks to the SDK, users can implement `handler` and `init` functions.
+
+## Init function
+
+This function receives a context that provides methods for logging and storing data that users
+can later use on the handler function. It doesn't return any value.
+
+`init` function is mainly used to initialize data, for example, loading models.
+
+### Python init example
+
+```python
+import joblib
+
+async def init(ctx):
+  ctx.logger.info("[worker init]")
+  ctx.set('model', joblib.load(ctx.path('models/model.joblib')))
+```
+
+### Golang init example
+
+```golang
+package main
+
+import (
+  "github.com/konstellation-io/kre/runners/kre-go"
+)
+
+func handlerInit(ctx *kre.HandlerContext) {
+  ctx.Logger.Info("init handler")
+    
+  // Saves a value in the context internal registry
+  ctx.Set("greeting", "Hello")   
+}
+```
+
+## Handler function
+
+This function receives a context, the same as the `init` function does, and the data, which is the incoming payload to process sent from this node's subscription. The context object is shared between different executions.
+
+`Handler` functions are used to process data, users can implement their service logic here.
+
+### Python handler example for runner compatible with KRT V1
```

Comment:
This whole section is confusing to me about how v1 and v2 are distinguished. Maybe separating the documentation of each version into different sections would help.
Something like:
## Runner SDK V1
### Init function
### Handler function
etc
## Runner SDK V2
### Init function
### Handlers
etc

Reply:
Good idea, I've added two subsections and refactored the SDK section